types/node/index.d.ts

```diff
@@ -44,13 +44,17 @@
 //                 ExE Boss <https://github.com/ExE-Boss>
 //                 Surasak Chaisurin <https://github.com/Ryan-Willpower>
 //                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
+//                 Jason Kwok <https://github.com/JasonHK>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
 
 // NOTE: These definitions support NodeJS and TypeScript 3.5.
 
 // NOTE: TypeScript version-specific augmentations can be found in the following paths:
 //          - ~/base.d.ts         - Shared definitions common to all TypeScript versions
 //          - ~/index.d.ts        - Definitions specific to TypeScript 2.8
+//          - ~/ts3.2/index.d.ts  - Definitions specific to TypeScript 3.2
```

Comment:
this lines looks dangerous for me. There are many options for node.js version, what we support.
This changes also add complexity for typescript version. As a result we have a matrix, what is too bit.

Reply:
Actually, "~/ts3.2" already exists before this pull request, it's just missing from tjat comment.

Also, the approach I used here is the result of TypeScript error `TS2403`, this error make me unable to simply override `global`.